torch/_dynamo/symbolic_convert.py

```diff
@@ -1102,6 +1102,7 @@ def step(self):
             return False
         self.current_instruction = inst = self.instructions[ip]
         self.instruction_pointer = ip + 1
+        counters["dynamo_step_count"]["total"] += 1
```

Comment:
Do we need to increment this every bytecode?  Why not just at the end and at jumps so the cost is not O(n).

Reply:
I think I updated this correctly, to do this at the end and when jumping :). (also let me know if you have good examples of code with jumps so I can help validate I did it correctly).